Desktop/MyBudgetWindow.cs

```diff
@@ -26,39 +27,98 @@ private void MyBudgetWindow_Load(object sender, EventArgs e)
 
         public void UpdateCategoriesList()
         {
-            CategoryList = (List<Category>)db.GetCategories();
+            _categoryList = (List<Category>) _db.GetCategories();
+            _transactionList = (List<Transaction>) _db.GetTransactions();
             PopulateCategoryListView();
+            Calculation(_categoryList, _transactionList);
         }
 
         private void PopulateCategoryListView()
         {
-            PopulateCategoryListView(CategoryList);
+            PopulateCategoryListView(_categoryList);
         }
 
-        private void PopulateCategoryListView(IEnumerable<Category> CategoryList)
+        public void Calculation (IEnumerable<Category> categoryList, IEnumerable<Transaction> transactionList)
         {
-            //this.budgetAndCategoriesView.Rows.Add("1", "Shopping", "30", "30", "0");
-            foreach (var category in CategoryList)
+            if (categoryList == null)
             {
-                budgetAndCategoriesView.Rows.Add(category.Id, category.Title, " ", " ");
+                throw new ArgumentNullException(nameof(categoryList));
+            }
+
+            if (transactionList == null)
+            {
+                throw new ArgumentNullException(nameof(transactionList));
+            }
+
+            decimal calc = 0;
+            int index = 0;
+           
+            foreach (var category in categoryList)
+            {
+                foreach (var transaction in transactionList)
+                {
+                    if (category.Id == transaction.CategoryId)
+                    {
+                        calc += transaction.Amount;
+                    }
+                }
+                this.budgetAndCategoriesView.Rows[index].Cells[3].Value = Math.Abs(calc);
```

Comment:
It is difficult to grasp which value is being assigned because `.Cells[3]` has no meaning.
But for now, it is ok since we will change the platform soon : )